tests/test_core.py

```diff
@@ -5237,6 +5237,7 @@ def test_netinet_in(self):
     self.do_run(src, expected)
 
   @needs_dlfcn
+  @no_wasm_backend('only tests tempDoublePtr alignment which does not exist on WASM backend')
```

Comment:
Looking in the test, that is only the second and more minor part of what it does, so I think we should only disable the tempDoublePtr part of that test (can ifdef for the wasm backend with `__wasm__`).

Reply:
If you look at `tests/core/test_main_module_static_align.txt`, it only contains:

    tempDoublePtr alignment: 0.

Seems like the test ultimately only checks for `tempDoublePtr` alignment, which is why I opted to disable the whole test.

Reply:
Oh, good catch! The
```
  emscripten_log(EM_LOG_WARN, "16-byte aligned char: %d.", int(&aligned));
```
should be an assertion that it is 16-byte aligned. I skimmed it too quickly :) Let's fix it that way, otherwise by removing the whole test we lose what partial testing we had with tempDoublePtr.